courses/factories.py

```diff
@@ -18,10 +18,18 @@ class CourseFactory(DjangoModelFactory):
     """Factory for Courses"""
     title = fuzzy.FuzzyText(prefix="Course ")
     program = factory.SubFactory(ProgramFactory)
+    position_in_program = factory.Sequence(lambda n: n)
 
     class Meta:  # pylint: disable=missing-docstring
         model = Course
 
+    @classmethod
+    def _setup_next_sequence(cls):
+        try:
+            return Course.objects.latest('position_in_program').position_in_program + 1
```

Comment:
It seems like `latest` is defined to be for date values: https://docs.djangoproject.com/en/1.9/ref/models/querysets/#latest. Can you use `order_by()` and `last()` instead?